firmware/screen/include/board/JC3248W535_GFX.h

```
@@ -0,0 +1,60 @@
+
+#include <Arduino_GFX_Library.h>
+
+// QSPI pins for the AXS15231B panel on this board.
+// CS, SCK, D0..D3 per the JC3248W535 schematic / community configs.
+#define JC_TFT_QSPI_CS   45
+#define JC_TFT_QSPI_SCK  47
+#define JC_TFT_QSPI_D0   21
+#define JC_TFT_QSPI_D1   48
+#define JC_TFT_QSPI_D2   40
+#define JC_TFT_QSPI_D3   39
+#define JC_TFT_RST       -1   // shared with board reset
```

Clang (14.0.6)

[warning] macro replacement list should be enclosed in parentheses

(bugprone-macro-parentheses)